app/post.go

```diff
@@ -220,7 +299,7 @@ func (a *App) CreatePost(post *model.Post, channel *model.Channel, triggerWebhoo
 	rpost = a.PreparePostForClient(rpost)
 
 	if err := a.handlePostEvents(rpost, user, channel, triggerWebhooks, parentPostList); err != nil {
-		return nil, err
+		mlog.Error("Failed to handle post events", mlog.Err(err))
```

Comment:
This is a logic change, as I didn't think it made sense to fail the call because of post-creation events when, in fact, the post was created successfully.